presto-main/src/test/java/io/prestosql/operator/TestDynamicFilterSourceOperator.java

```diff
@@ -280,44 +362,77 @@ public void testCollectEmptyBuildSide()
     public void testCollectTooMuchRows()
     {
         int maxRowCount = getDynamicFilteringMaxPerDriverRowCount(pipelineContext.getSession());
-        Page largePage = createSequencePage(ImmutableList.of(BIGINT), maxRowCount + 1);
+        Page largePage = new Page(createLongSequenceBlock(0, maxRowCount + 1),
+                createLongSequenceBlock(maxRowCount + 1, 2 * (maxRowCount + 1)));
 
-        OperatorFactory operatorFactory = createOperatorFactory(channel(0, BIGINT));
+        OperatorFactory operatorFactory = createOperatorFactory(channel(0, BIGINT), channel(1, BIGINT));
         verifyPassthrough(createOperator(operatorFactory),
-                ImmutableList.of(BIGINT),
+                ImmutableList.of(BIGINT, BIGINT),
```

Comment:
why do we need another column?

Reply:
Multiple columns to verify that min/max of each column is being maintained as expected separately

Reply:
Separated into 2 tests `testCollectTooMuchRowsSingleColumn` and `testCollectTooMuchRowsMultipleColumns` now